### PR TITLE
Add functionality to delete comments.

### PR DESCRIPTION
**Pull request by zrideaux**
+ Create delete all comments button and function.
+ Add entity keys to the JSON data, allowing for specific comments to be altered.
+ Replace in-comment delete link placeholders with functioning delete buttons.
+ Minor style tweaks.
  + Add drop shadow to sidebar on portrait devices.
  + Improve style on comments page.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -117,16 +117,18 @@ <h1>Comments</h1>
...
+            <select name="show" id="number-of-comments" type="number" onchange="getCommentsFromServlet();" placeholder="ex. 5">
               <option value="5" selected="selected">5 Comments</option>
               <option value="10">10 Comments</option>
               <option value="20">20 Comments</option>
               <option value="-1">All Comments</option>
-          </select>
-        </form>
+            </select>
+          </form>
+          <button action="POST" id="delete-all" onclick="deleteAllComments();">Delete All Comments</button>
```
Comment (hoorayhu) [inline portfolio/src/main/webapp/index.html:130]: No need for the action attribute.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -117,16 +117,18 @@ <h1>Comments</h1>
           <br>
           <input type="submit"></input>
         </form>
-        <hr>
-        <form>
-          <label>Comments Displayed:</label>
-          <select name="show" id="number-of-comments" type="number" onchange="getCommentsFromServlet();" placeholder="ex. 5">
+        <div id="comment-actions">
+          <form>
```
Comment (hoorayhu) [inline portfolio/src/main/webapp/index.html:121]: You don’t need the form any more.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (zrideaux) [reply portfolio/src/main/webapp/index.html:130]: Removed!

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (zrideaux) [reply portfolio/src/main/webapp/index.html:121]: And removed!

**Pull request merged**